zulipterminal/ui.py

```diff
@@ -77,6 +77,26 @@ def set_footer_text(self, text_list: Optional[List[Any]]=None,
             time.sleep(duration)
             self.set_footer_text()
 
+    @asynch
+    def set_typeahead_footer(self, suggestions: List[Any],
```

Comment:
You loosened this type compared to the last iteration? What's the reasoning?

Reply:
I can't make this `List[str]` because I need to do assign a tuple for highlighting
```
suggestions[state] = ('code', suggestions[state])
```

Reply:
The older version was `List[Tuple[str, str]]` which didn't cause problems here.

Reply:
Right, but from a logical perspective, we just know we have a `List[str]` coming in. You don't *have* to assign to the same name, and perhaps it would be clearer if you didn't? It doesn't really make much sense to have the not-found case be 'suggestions' by name, either.

Reply:
When I assign it to a new name, I would still need a `List[Any]` type annotation?

So the best place to do it would be 
```python3
footer_text = list(map(lambda x: ' ' + x + ' ', suggestions))
```
But the type annotation will not fit in this line, so I'd end up doing
```python3
footer_text = suggestions  # type: List[Any]
```
Is it really worth adding a new line to copy a variable just to typecast it?